### PR TITLE
duncrow-gmbh/parallax

### PR DESCRIPTION
**Pull request by cmartin6**
updated fork, added duncrow-gmbh/parallax

### CODE REVIEW

File: linter/whitelists/default.txt [hunk 1]
```diff
@@ -124,6 +124,8 @@ OSRM
 Overpass
 Overpass
 Open
+Parallax
+parallax
```
Comment (fritzmg) [inline linter/whitelists/default.txt]: These should not be in the default whitelist, as they are ordinary English words.

File: linter/whitelists/de.txt [hunk 2]
```diff
@@ -296,6 +296,8 @@ Paginierungen
 Panel
 Panelen
 Panels
+Parallax
+parallax
```
Comment (xchs) [inline linter/whitelists/de.txt]: I would have added these words to the English whitelist, but, well, okay.

Comment (cmartin6) [reply linter/whitelists/de.txt]: But they are used in the german translation as well?

Comment (fritzmg) [reply linter/whitelists/de.txt]: >  would have added these words to the English whitelist, but, well, okay.

They don't need to be, unless the spell checker flagged them?

Comment (fritzmg) [reply linter/whitelists/de.txt]: Btw. you only use `Parallax` in your German meta data, not `parallax`. The latter should be removed.

**Pull request merged**